applications/nrf5340_audio/doc/user_interface.rst

```diff
@@ -0,0 +1,162 @@
++---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+| **PLAY/PAUSE**| Starts or pauses the playback.                                                                            | All                                         |
++---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+| **BTN 4**     | Long-pressed during startup: Turns on the DFU mode, if                                                    | All                                         |
+|               | the device is :ref:`configured for it<nrf53_audio_app_configuration_configure_fota>`.                     |                                             |
+|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|               | Pressed on the gateway during playback: Sends a test tone generated on the device.                        | * :ref:`nrf53_audio_broadcast_source_app`   |
+|               | Use this tone to check the synchronization of headsets.                                                   | * :ref:`nrf53_audio_unicast_client_app`     |
+|               +-----------------------------------------------------------------------------------------------------------+                                             |
+|               | Pressed on the gateway during playback multiple times: Changes the tone frequency.                        |                                             |
+|               | The available values are 1000 Hz, 2000 Hz, and 4000 Hz.                                                   |                                             |
+|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|               | Pressed on a BIS headset during playback: Change audio stream, if more than one is available.             | :ref:`nrf53_audio_broadcast_sink_app`       |
++---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+| **BTN 5**     | Long-pressed during startup: Clears the previously stored bonding information.                            | All                                         |
+|               +-----------------------------------------------------------------------------------------------------------+                                             |
+|               | Pressed during playback: Mutes the playback volume.                                                       |                                             |
```

Comment:
It's not muting for all devices, since broadcast sink will change gateway when BTN5 is pressed and broadcast source doesn't use BTN5

Reply:
These should be listed as muting
* :ref:`nrf53_audio_unicast_server_app` ,
* :ref:`nrf53_audio_unicast_client_app`